File: app/src/main/java/com/bdcoe/saksham/POJOs/Schedule/List.java

package com.bdcoe.saksham.POJOs.Schedule;

import com.google.gson.annotations.SerializedName;

public class List {

    @SerializedName("date")
    private String mDate;
    @SerializedName("sport")
    private String mSport;
    @SerializedName("teams")
    private String mTeams;
    @SerializedName("time")
    private String mTime;

    public String getDate() {
        return mDate;
    }

    public void setDate(String date) {
        mDate = date;
    }

    public String getSport() {
        return mSport;
    }

    public void setSport(String sport) {
        mSport = sport;
    }

    public String getTeams() {
        return mTeams;
    }

    public void setTeams(String teams) {
        mTeams = teams;
    }

    public String getTime() {
        return mTime;
    }

    public void setTime(String time) {
        mTime = time;
    }

}

File: app/src/main/java/com/bdcoe/saksham/POJOs/News/List.kt
package com.bdcoe.saksham.POJOs.News

import com.google.gson.annotations.SerializedName


class List {

    @SerializedName("classified_id")
    var classifiedId: String? = null
    @SerializedName("desc")
    var desc: String? = null
    @SerializedName("image1")
    var image1: String? = null
    @SerializedName("image2")
    var image2: String? = null
    @SerializedName("sport")
    var sport: String? = null
    @SerializedName("teams")
    var teams: String? = null
    @SerializedName("timestamp")
    var timestamp: String? = null

}
